src/modules/utils/blockExplorer/blockExplorerUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { type Chain } from 'wagmi/chains';
+
+class BlockExplorerUtils {
+    getAddressUrl(address: string, network: Chain): string {
```

Comment:
Can't we just pass the `chainId` and get the chain informations from `viem` instead? I would also handle a undefined `chainId` to avoid always checking on the components if the `chainId` is defined

Reply:
@cgero-eth, how do you suggest we handle an `undefined` `chainId`? Should we default to Etherscan if no `chainId` is provided?

Reply:
@rcaragon Yes I'd just default the chainId to 1